### DISCUSSION TITLE
how change base url from swagger

### DISCUSSION THREAD

**Discussion by FelipeVeiga** [dimaskiddo/go-whatsapp-multidevice-rest#38] [Q&A] [answered]
hello there.
how can i change base url from swagger in the docker image ?
I mean the IP address and not the path

thanks a lot.

**Comment by dimaskiddo** [comment, marked as answer]
Hai @FelipeVeiga sorry for my late response in this discussion, thank for your findings
I already update the swago how to communicate between the Host and BasePath by this commit 1df9038
This commit also released in version [v1.6.1](https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/releases/tag/v1.6.1)

I hope you can help me to test in in docker also if this Dynamic Host and Dynamic Base Path is working for OpenAPI / Swagger
Thank you so much 😄 🥂
